cmd/lifecycle/exporter.go

```diff
@@ -57,7 +57,8 @@ func (e *exportCmd) DefineFlags() {
 	cli.FlagStackPath(&e.StackPath)
 	cli.FlagUID(&e.UID)
 	cli.FlagUseDaemon(&e.UseDaemon)
-
+	cli.FlagLayoutDir(&e.LayoutDir)
+	cli.FlagUseLayout(&e.UseLayout)
```

Comment:
We should guard this on the platform API being at least... 0.12? Not sure when this will land...